staging/src/k8s.io/pod-security-admission/policy/check_procMount_test.go

```diff
@@ -29,10 +29,12 @@ func TestProcMount(t *testing.T) {
 
 	hostUsers := false
 	tests := []struct {
-		name         string
-		pod          *corev1.Pod
-		expectReason string
-		expectDetail string
+		name                                     string
+		pod                                      *corev1.Pod
+		expectReason                             string
+		expectDetail                             string
+		allowed                                  bool
```

Comment:
```suggestion
		expectAllowed                                  bool
```

Reply:
done, as well as did so in other unit tests (which I based handling of this on)